docs/sources/alerting/notifications.md

```diff
@@ -185,6 +185,12 @@ Notifications can be sent by setting up an incoming webhook in Google Hangouts c
 
 Squadcast helps you get alerted via Phone call, SMS, Email and Push notifications and lets you take actions on those alerts. Grafana notifications can be sent to Squadcast via a simple incoming webhook. Refer the official [Squadcast support documentation](https://support.squadcast.com/docs/grafana) for configuring these webhooks.
 
+### Prometheus Alertmanager
+
+Alertmanager handles alerts sent by client applications such as Prometheus server or Grafana. It takes care of deduplicating, grouping, and routing them to the correct receiver. Grafana notifications can be sent to Alertmanager via a simple incoming webhook. Refer the official [Prometheus Alertmanager documentation](https://prometheus.io/docs/alerting/alertmanager) for configuration.
+
+**Caution**: In case of an high availability setup, it's important to not load balance traffic between Grafana and Alertmanagers, but instead, point Grafana to a list of all Alertmanagers, by specify their URLs comma-separated in the notification channel configuration.
```

Comment:
```suggestion
> **Caution:** In case of a high-availability setup, do not load balance traffic between Grafana and Alertmanagers. Instead, point Grafana to a list of all Alertmanagers, by specify their URLs comma-separated in the notification channel configuration.
```

Reply:
Why is this a caution? What bad thing will happen if they ignore this instruction?